cached_downloader.go

```diff
@@ -22,8 +22,8 @@ type CachedDownloader interface {
 	Fetch(urlToFetch *url.URL, cacheKey string, transformer CacheTransformer, cancelChan <-chan struct{}) (io.ReadCloser, int64, error)
 }
 
-func NoopTransform(source, destination string) (int64, error) {
-	err := os.Rename(source, destination)
+func NoopTransform(source string, destination string) (int64, error) {
```

Comment:
It's idiomatic Go to collapse type declarations for adjacent arguments of the same type, as was done before the change.